akka-persistence/src/main/scala/akka/persistence/JournalProtocol.scala

```diff
@@ -142,3 +142,13 @@ private[persistence] object JournalProtocol {
   final case class ReplayMessagesFailure(cause: Throwable) extends Response with DeadLetterSuppression
 
 }
+
+/**
+  * Reply message to a successful [[JournalProtocol.DeleteMessagesTo]] request.
+  */
+final case class DeleteMessagesSuccess(toSequenceNr: Long) extends JournalProtocol.Response
+
+/**
+  * Reply message to a failed [[JournalProtocol.DeleteMessagesTo]] request.
+  */
+final case class DeleteMessagesFailure(cause: Throwable, toSequenceNr: Long) extends JournalProtocol.Response
```

Comment:
No risk that changing the hierarchy here breaks existing journals - it is only used between our journal logic an the persistent actors/eventsourced actors?

Reply:
I saw no evidence.